### PR TITLE
Format query

### PR DESCRIPTION
**Pull request by icreated**

### CODE REVIEW

File: src/test/kotlin/com/cosmotech/api/utils/StringExtensionsTests.kt [hunk 1]
```diff
@@ -40,4 +40,20 @@ class StringExtensionsTests {
     val actual = input.toRedisMetaDataKey()
     assertEquals(expected, actual)
   }
+
+  @Test
+  fun `should format Cypher Query with variables`() {
+    val map = mapOf("name" to "Joe Doe", "id" to "100", "object" to "{id:\"id\"}")
```
Comment (jreynard-code) [inline src/test/kotlin/com/cosmotech/api/utils/StringExtensionsTests.kt:46]: Could you please add a test with an array parameter please?

**Review by jreynard-code — CHANGES_REQUESTED** [review]
Just some remarks about syntactic sugar and  add a test case with array property ;)

**Review by icreated — COMMENTED** [review]
Done

**Review by jreynard-code — APPROVED** [review]
LGTM :+1:

**Pull request merged**